hni-admin-service/src/main/java/org/hni/admin/service/UserOnboardingController.java

```diff
@@ -212,5 +213,17 @@ public Response isUserNameAvailable(Map<String, String> userNameInfo) {
 		}
 		return Response.ok(response).build();
 	}
+	
+	@GET
+	@Path("/getAllNgo")
+	@Produces({ MediaType.APPLICATION_JSON })
+	@ApiOperation(value = "", notes = "", response = List.class, responseContainer = "")
+	public List<NgoBasicDto> getAllNgo() {
+		

```

Comment:
Please give proper descriptions to methods on swagger, a try catch and valid loggers needs to be added here.